File: add/codesnippet/CPP/p-system.windows.forms.i_27_1.cpp
public:
   void MyHandle()
   {
      // Gets the current input language.
      InputLanguage^ myCurrentLanguage = InputLanguage::CurrentInputLanguage;
      
      // Gets a handle for the language and prints the number.
      IntPtr myHandle = myCurrentLanguage->Handle;
      textBox1->Text = String::Format( "The handle number is: {0}", myHandle );
   }

File: add/codesnippet/VisualBasic/p-system.windows.forms.i_27_1.vb
 Public Sub MyHandle()
    ' Gets the current input language.
    Dim myCurrentLanguage As InputLanguage = InputLanguage.CurrentInputLanguage
        
    ' Gets a handle for the language  and prints the number.
    Dim myHandle As IntPtr = myCurrentLanguage.Handle
    textBox1.Text = "The handle number is: " & myHandle.ToString()
 End Sub
